apps/todo-list-client/src/app/check-box-button/check-box-button.component.html

```diff
@@ -1,14 +1,19 @@
 <div>
   <label class="relative flex cursor-pointer items-center rounded-full p-3" for="checkbox-8" data-ripple-dark="true">
-    <input
-      type="checkbox"
-      class="before:content[''] border-blue-gray-200 before:bg-blue-gray-500 peer relative h-5 w-5 cursor-pointer appearance-none rounded-md border bg-gray-400 transition-all before:absolute before:left-2/4 before:top-2/4 before:block before:h-12 before:w-12 before:-translate-x-2/4 before:-translate-y-2/4 before:rounded-full before:opacity-0 before:transition-opacity checked:border-blue-500 checked:bg-blue-500 checked:before:bg-blue-500 hover:before:opacity-10"
+    <button
+      type="button"
+      class="before:content[''] border-blue-gray-200 before:bg-blue-gray-500 peer relative h-5 w-5 cursor-pointer appearance-none rounded-md border transition-all before:absolute before:left-2/4 before:top-2/4 before:block before:h-12 before:w-12 before:-translate-x-2/4 before:-translate-y-2/4 before:rounded-full before:opacity-0 before:transition-opacity hover:before:opacity-10"
```

Comment:
Could you please normalize this list of styles? I mean you can customize various aspects of Tailwind...
Ex:

`module.exports = {
  theme: {
    extend: {
      button-colors: {
        primary: '#00000',
        secondary: '#11111',
      }
      font: {
        'font-style': 'Arial',
        'size': '12rem',
      },
    },
  },
  variants: {},
  plugins: [],
}`